sdks/java/harness/jmh/src/main/java/org/apache/beam/fn/harness/jmh/control/ExecutionStateSamplerBenchmark.java

```diff
@@ -103,33 +104,38 @@ public void tearDown() {
   }
 
   @Benchmark
-  @Threads(1)
-  public void testTinyBundleRunnersCoreStateSampler(RunnersCoreStateSampler state)
+  @Threads(10)
+  @Fork(1)
+  public void testTinyBundleRunnersCoreStateSampler(RunnersCoreStateSampler state, Blackhole bh)
```

Comment:
Did you see that we needed the blackhole?

Reply:
I figured it'd be better to be safe than sorry, it also adds a little non-trivial work to the loop which is nice for the benchmark.

Reply:
sgtm